crates/ruff_python_parser/src/token.rs

```diff
@@ -268,7 +268,7 @@ impl fmt::Display for Tok {
             Rsqb => f.write_str("']'"),
             Colon => f.write_str("':'"),
             Comma => f.write_str("','"),
-            Comment(value) => f.write_str(value),
+            Comment => f.write_str("Comment"),
```

Comment:
I'm worried that emitting `Comment` here is confusing. I'm not sure what a better value is; maybe I'm making it clear that it is a placeholder `<Comment>` and not the actual `Comment`?